setup.py

```diff
@@ -24,7 +24,7 @@
 
 
 PACKAGE = {
-    'version': '0.36',
+    'version': '0.36.1',
```

Comment:
i don't like to switch numbering scheme. please use `0.37`

Reply:
No. We use patch level numbers everywhere else as well. This does not warrant a minor version update at all.

Reply:
yes, but we weren't here before, so no reason to start now.